src/main/java/rs/teslaris/core/indexmodel/DocumentFileIndex.java

```diff
@@ -0,0 +1,47 @@
+package rs.teslaris.core.indexmodel;
+
+import javax.persistence.Id;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.springframework.data.elasticsearch.annotations.Document;
+import org.springframework.data.elasticsearch.annotations.Field;
+import org.springframework.data.elasticsearch.annotations.FieldType;
+import org.springframework.data.elasticsearch.annotations.Setting;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Document(indexName = "document_file")
+@Setting(settingPath = "/configuration/serbian-analyzer-config.json")
+public class DocumentFileIndex {
+
+    @Id
+    private String id;
+
+    @Field(type = FieldType.Text, store = true, name = "pdf_text_srp", analyzer = "serbian_analyzer", searchAnalyzer = "serbian_analyzer")
```

Comment:
if english is used as term for english analyzer, then we should rename serbian_analyzer to serbian